### PR TITLE
Psalm v5

### PR DESCRIPTION
**Pull request by msmakouz**
| Q             | A
| ------------- | ---
| Bugfix?       | ❌
| Breaks BC?    | ❌ 
| New feature?  | ❌ 

The RedundantCondition error with reflections is most likely related to the: https://github.com/vimeo/psalm/issues/9489

### CODE REVIEW

**Review by butschster — APPROVED** [review]

File: src/Http/src/Request/FilesBag.php [hunk 1]
```diff
@@ -13,7 +13,6 @@
  * @method UploadedFileInterface|null get(int|string $name, $default = null)
  * @method UploadedFileInterface[] all()
  * @method UploadedFileInterface[] fetch(array $keys, bool $fill = false, $filler = null)
- * @method \Traversable<array-key, UploadedFileInterface> getIterator()
```
Comment (roxblnfk) [inline src/Http/src/Request/FilesBag.php:16]: Need to replace with `@inplements IteratorAggregate<...>`

File: src/Stempler/src/Node/Aggregate.php [hunk 2]
```diff
@@ -79,10 +79,9 @@ public function accepts(string $name): ?string
         }
 
         return null;
-    }
-
-    /**
-     * @return \Generator<string, TNode[]>
+    }/**
```
Comment (roxblnfk) [inline src/Stempler/src/Node/Aggregate.php:82]:
```suggestion
    }

    /**
```

**Review by roxblnfk — COMMENTED** [review]
Annotations `@template-implements` and `@implements` are equals.
The same about `@template-extends` and `@extends`

File: src/Stempler/src/Node/Aggregate.php (hunk 2 above)
Comment (msmakouz) [reply src/Stempler/src/Node/Aggregate.php]: Fixed

File: src/Http/src/Request/FilesBag.php (hunk 1 above)
Comment (msmakouz) [reply src/Http/src/Request/FilesBag.php:16]: Added @inplements IteratorAggregate

File: src/Core/src/BinderInterface.php [hunk 3]
```diff
@@ -44,7 +42,7 @@ public function removeBinding(string $alias): void;
      * @template TClass of object
      *
      * @param class-string<TClass> $class
-     * @param class-string<InjectorInterface<TClass>> $injector
+     * @param class-string $injector
```
[the remark this answers is not in the document]
Comment (msmakouz) [reply src/Core/src/BinderInterface.php]: Related to this issue: https://github.com/vimeo/psalm/issues/8810
Reverted and added suppress.

**Pull request merged**